.github/workflows/compare-layouts.yml

```
@@ -0,0 +1,244 @@
+        run: npm install @openzeppelin/upgrades-core
+      - name: Compare the layouts
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        id: compare-layouts
+        run: |
+          node script/compareLayouts.js
+        # Even if this fails, the CI status should be updated.
+        continue-on-error: true
+      - name: Update parent commit status
+        if: always()
+        # if the outcome is not set, it will post failure
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2015:info:7:61: Note that A && B || C is not if-then-else. C may run when A is true

(shellcheck)